app/lib/search/index_simple.dart

```diff
@@ -168,6 +161,81 @@ class SimplePackageIndex implements PackageIndex {
       value: (String url) => _documents[url].popularity * 100,
     );
   }
+
+  Score _searchText(String text, String packagePrefix) {
+    if (text != null && text.isNotEmpty) {
+      final Score textScore = new Score()
+        ..addValues(_nameIndex.search(text), 0.82)
+        ..addValues(_descrIndex.search(text), 0.12)
+        ..addValues(_readmeIndex.search(text), 0.06);
+      // removes scores that are less than 5% of the best
```

Comment:
This comment is confusing, since `less than 5% of the best` could be interpreted as we drop the lowest `95%`. Maybe just consider removing these comments if the code itself is self-explaining.